opt_out_http_api/api_methods.py

```diff
@@ -118,3 +120,8 @@ def delete_address(self, request, addresstype, address):
         if opt_out is None:
             raise OptOutNotDeleted()
         return self.response(request, opt_out=opt_out)
+
+    @app.route('/optouts/count', methods=['GET'])
+    def count_address(self, request):
```

Comment:
We should perhaps rename this to `count_opt_outs` since we're counting opt outs and not addresses.